irods_consortium_continuous_integration_build_hook.py

```diff
@@ -15,9 +15,9 @@ def add_cmake_to_front_of_path():
 
 def install_building_dependencies(externals_directory):
     externals_list = [
-        'irods-externals-boost1.81.0-0',
-        'irods-externals-clang-runtime13.0.0-0',
-        'irods-externals-clang13.0.0-0',
+        'irods-externals-boost-libcxx1.81.0-1',
```

Comment:
@SwooshyCueb Is this the correct externals pkg for main?

Looking at the following lines from irods/irods.
- https://github.com/irods/irods/blob/bc2a9bf0d8d3587d1aad8bf6a9b3db650e0d6944/CMakeLists.txt#L25-L43

Reply:
No, main should have non-libcxx packages, and should omit clang-runtime.